Resources/views/js.html.twig

```diff
@@ -23,6 +23,13 @@
             this.language = '{{ language }}';
         {%- endif -%}
 
+        {%- if callback is defined -%}
+            var fn = window['{{callback}}'];
+            if(typeof fn === 'function') {
```

Comment:
I think I'd like to throw an exception if this is NOT a function - because it means the developer messed something up